lib/components/viewers/RouteRow.js

```diff
@@ -39,7 +39,8 @@ export const OperatorImg = styled.img`
 `
 
 export const ModeIconElement = styled.span`
-  height: 22px;
+  height: 28px;
```

Comment:
This is slightly interfering with how the route viewer for configs that use smaller icons (the bus icon can have too much offset wrt to the text, see screnshot):

<img width="151" alt="image" src="https://user-images.githubusercontent.com/56846598/225363998-b3846407-71d9-4bf5-9a8f-9db6cf4b5d5d.png">

Reply:
Great catch! Thanks for testing all cases. Fixed in https://github.com/opentripplanner/otp-react-redux/pull/824/commits/3f0b7eeed81fc7cab19f793e3a9ef9219f16bcff
<img width="89" alt="image" src="https://user-images.githubusercontent.com/62163307/225744282-2a2879f7-25bc-4528-88b2-801db6f14b9d.png">